libraries/AP_Common/missing/cmath

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <toolchain.h>
```

Comment:
I disagree as that doesn't make sense **to me**. It would if the definition was passed through command line, which would make it hard for the build system to know which sources actually depend on a certain define.